src/blocks/sound.rs

```diff
@@ -673,12 +901,19 @@ impl Block for Sound {
             if name.as_str() == self.id {
                 match e.button {
                     MouseButton::Right => self.device.toggle()?,
-                    MouseButton::Left =>
```

Comment:
What's the reason for moving away from `spawn_child_async()` here?

Reply:
Can you then rebase your pull request?